### PR TITLE
task/open_0.1.1

### PR DESCRIPTION
**Pull request by frbattid**

### CODE REVIEW

File: cosmos-auth/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "cosmos-auth",
   "description": "Node.js app exposing Cosmos Auth API",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (fgalan) [inline cosmos-auth/package.json]: Shouldn't it be 0.1.0-next or 0.1.0_SNAPSHOT instaed of a concret number such as "0.1.1"? At least that is the criteria used in other FIWARE components...

Comment (fgalan) [inline cosmos-auth/package.json]: As in https://github.com/telefonicaid/perseo-fe/blob/develop/package.json#L4 (another Node.js project using that approach)

Comment (frbattid) [inline cosmos-auth/package.json]: Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/0026c17b38708be56dd7c986e84f62ddb02df7cd

**Comment by fgalan** [comment]
LGTM

**Pull request merged**